tests/e2e/system_test/test_reclaim_space_cronjob.py

```diff
@@ -0,0 +1,212 @@
+import logging
+import pytest
+import yaml
+from uuid import uuid4
+import os
+import time
+from tempfile import NamedTemporaryFile
+
+
+from ocs_ci.framework.testlib import (
+    skipif_ocp_version,
+    skipif_ocs_version,
+    ManageTest,
+    tier3,
+)
+from ocs_ci.utility.utils import run_cmd
+from ocs_ci.helpers.performance_lib import run_oc_command
+from ocs_ci.helpers import helpers
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.exceptions import PVCNotCreated
+
+logger = logging.getLogger(__name__)
+ERRMSG = "Error in command"
+
+
+@tier3
+@skipif_ocp_version("<4.14")
+@skipif_ocs_version("<4.14")
+class TestReclaimSpaceCronJob(ManageTest):
+    """
+    Test that verifies creation of Reclaim Space Cron Jobs for RBD PCSs in openshift-* namespaces
```

Comment:
Automatically?

Reply:
Yes, automatically. Will add to the description.